website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
@samchrisinger ObjectID+JWT would be a great implementation.

@HarryRybacki While they do already exist, the obvious purpose is a temporary state--I can't imagine a situation where someone would link to those. Anyone have thoughts? I need to know a bit more about these routes. Will discuss in person.